lib/amazon-pricing/definitions/instance-type.rb

```diff
@@ -433,12 +451,18 @@ def self.get_values(json, category_type, override_price = false)
       'c5.4xlarge'  => [ 281, 16000], # EBSOptimized
       'c5.9xlarge'  => [ 563, 32000], # EBSOptimized
       'c5.18xlarge' => [1125, 64000], # EBSOptimized
-      'c5d.large'    => [2250, 16000], # NVMe
-      'c5d.xlarge'   => [2250, 16000], # NVMe
-      'c5d.2xlarge'  => [2250, 16000], # NVMe
-      'c5d.4xlarge'  => [2250, 16000], # NVMe
-      'c5d.9xlarge'  => [4500, 32000], # NVMe
-      'c5d.18xlarge' => [9000, 64000], # NVMe
+      'c5d.large'    => [437, 20000], # NVMe
+      'c5d.xlarge'   => [437, 20000], # NVMe
+      'c5d.2xlarge'  => [437, 20000], # NVMe
+      'c5d.4xlarge'  => [437, 20000], # NVMe
+      'c5d.9xlarge'  => [875, 40000], # NVMe
+      'c5d.18xlarge' => [1750, 80000], # NVMe
```

Comment:
was our original c5d performance incorrect, I presume?

Reply:
Yep :/ Eyeballing it quickly, that was the only one I saw.